tests/backup/scheduled_test.js

```diff
@@ -336,18 +338,15 @@ Scenario('PMM-T901 Verify user can delete scheduled backup @backup @bm-mongo',
     await scheduledAPI.createScheduledBackup(schedule);
     await scheduledPage.openScheduledBackupsPage();
 
-    // Open Delete modal
-    I.click(scheduledPage.buttons.deleteByName(schedule.name));
+    scheduledPage.openDeleteModal(schedule.name);
 
     // Click Cancel button and verify schedule still exists
-    I.waitForVisible(scheduledPage.buttons.confirmDelete, 10);
     I.click(scheduledPage.buttons.cancelDelete);
     I.dontSeeElement(scheduledPage.elements.modalContent);
-    I.seeElement(scheduledPage.buttons.deleteByName(schedule.name));
+    I.seeElement(scheduledPage.buttons.actionsMenuByName(schedule.name));
```

Comment:
waitForVisible

Reply:
added. though for this very line wait is not required - it's reasonable for the previous check.